src/types.tsx

```diff
@@ -119,7 +119,7 @@ export interface UserProfile {
 
 export interface DiscussionResponse {
   status: string;
-  page: number;
+  currentPage: number;
```

Comment:
Is this related to moving the file?

Reply:
I assume the fixtures come from actual API calls. If so there does not exist a `page` key only `currentPage`, so i made the relavant changes